components/Cards/card.css

```diff
@@ -0,0 +1,231 @@
+.cards-wrapper {
+    background-color:var(--white);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    justify-self: center;
+    align-self: center;
+    padding: 2rem;
+    width: fit-content;
+    margin: 2rem;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.card,
+.card-basic {
+    width: 22rem;
+    height: 28rem;
+    background-color: var(--secondary-color);
+    border-radius: 0.5rem;
+    -webkit-border-radius: 0.5rem;
+    -moz-border-radius: 0.5rem;
+    -ms-border-radius: 0.5rem;
+    -o-border-radius: 0.5rem;
+    box-shadow: -3px 3px 3px gray;
+}
+
+.card:hover {
+    box-shadow: -3px 3px 3px black;
+}
+
+.image-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+
+.image-container img {
+    width:22rem;
+    height:15rem;
+    max-inline-size: 100%;
+    border-radius: 0.5rem 0.5rem 0 0;
+    -webkit-border-radius: 0.5rem 0.5rem 0 0;
+    -moz-border-radius: 0.5rem 0.5rem 0 0;
+    -ms-border-radius: 0.5rem 0.5rem 0 0;
+    -o-border-radius: 0.5rem 0.5rem 0 0;
+}
+
+.content-container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 0.5rem;
+}
+
+.content-container h1 {
+    font-size: 1.5rem;
+    font-weight: 500;
+}
+
+.content-container h3 {
+    font-size: 1rem;
+    color: var(--dark-gray);
+    font-weight: 400;
+}
+
+.content-container p {
+    padding-top: 1rem;
+    color: var(--light-gray);
+    font-size: 0.9rem;
+}
+
+.action-items-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    width: fit-content;
+}
+
+.btn-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: flex-start;
```

Comment:
Here as well, `flex-direction` `align-items` `justify-content` you're giving the default value which could be avoided for reducing the lines of code.

Reply:
Done.